sci-electronics/drahnr-oregano/files/drahnr-oregano-0.84.39-fix-xdg-gschema.patch

```diff
@@ -0,0 +1,13 @@
+--- oregano-0.84.39/wscript.orig	2018-08-21 01:24:38.938484249 +0300
++++ oregano-0.84.39/wscript	2018-08-21 01:22:41.453484758 +0300
+@@ -18,8 +18,8 @@
+ 
+ 	opt.add_option('--debug', dest='build_debug', action='store_true', default=False, help='Build with debug flags')
+ 	opt.add_option('--release', dest='build_release', action='store_true', default=False, help='Build with release flags')
+-	opt.add_option('--no-install-gschema', dest='no_install_gschema', action='store_true', default=False, help='Do not install the schema file')
+-	opt.add_option('--no-xdg', dest='no_xdg', action='store_true', default=False, help='Do not install the xdg desktop or xdg mime files')
++	opt.add_option('--no-install-gschema', dest='no_install_gschema', action='store_true', default=True, help='Do not install the schema file')
```

Comment:
Don't patch the wscript. Instead pass appropriate options to waf. waf-utils has some docs on how to do it.

Reply:
"waf install" has no option on waf.eclass. options passed via "waf configure" not saved, i checked it. i tryed read waf's doc but can't see answer - seems i must work, but not work. ok, i'll wrote to upstream...

Reply:
That sucks. Maybe we could just enhance waf-utils to pass `"${@}"` over in all phases. Alternatively, you could just call `"${WAF_BINARY}" --destdir="${D}" install` directly — it doesn't really look that special.

Reply:
Actually, this is an impl detail on the `wscript` if/how those options are stored. I would recommend to wait until the waf binary is updated since behaviour might change.

Reply:
> Ping.

Wait two days, upsteam returned from vacation.

Reply:
@ktrace how did you check the options were not saved?

Reply:
still not saved. I will workaround it. and maybe need up tag to 0.84.40?
python3.7 not checked yet, but plan in two days.

Reply:
> Maybe we could just enhance waf-utils to pass `"${@}"` over in all phases.

i think it'll be great